Sprint-1/interpret/to-pounds.js

```diff
@@ -25,3 +25,11 @@ console.log(`£${pounds}.${pence}`);
 
 // To begin, we can start with
 // 1. const penceString = "399p": initialises a string variable with the value "399p"
+// 2. const penceStringWithoutTrailingP = penceString.substring(0,penceString.length - 1) : initialises 399 to penceStringWithoutTrailingP
+//    substring extract string part so starting from 0 index and excluding end index ie, 3 (0, 4 - 1); it removes last character 'p'
+// 3. const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"): padStart adds 0 from the start if length is less than 3
+// it make sure that string is having atleast 3 characters to split into pounds and pence, here '399' already have length 3 so its gives 399   
+// 4. const pounds = paddedPenceNumberString.substring(0, paddedPenceNumberString.length - 2): extracts substring 3 pound part from a string 399
+// 5. const pence = paddedPenceNumberString.substring(paddedPenceNumberString.length - 2).padEnd(2, "0"); extract pence part that is 99 here from 
+//    string 399 and ensures it as atleast 2 characters in pence by using padEnd method
```

Comment:
Do you think the code will still work correctly if we remove ` .padEnd(2, "0")` from line 16?

Reply:
For "399", .padEnd(2, "0") is not necessary since pence part "99" already has 2 characters. If we remove, padEnd(2, "0") it might result in incorrect formatting if the penceString length is less than 3 characters. (for Eg: "39")

Reply:
Well, did you test the code to see if it meet your expectation?